packages/faucet/src/faucet.spec.ts

```diff
@@ -81,7 +75,34 @@ describe("Faucet", () => {
   });
 
   describe("send", () => {
-    it("can send", async () => {
+    it("can send bank token", async () => {
```

Comment:
So nice..

Reply:
Oh, I mixed up the titles. This is the contract test, not the bank test